phase1/vsphere/README.md

```diff
@@ -75,40 +72,50 @@ If you do power it on, future clones of this VM will end up [getting the same IP
 echo -n > /etc/machine-id
 ```
 
+### Launch deployment environment
 
-### Download `kubernetes-anywhere`:
+#### Pull `cnastorage/kubernetes-anywhere:latest` docker image:
 
 ```shell
-curl -sL https://github.com/kubernetes/kubernetes-anywhere/archive/v0.1.0.tar.gz | tar xz
-cd kubernetes-anywhere-0.1.0
+docker pull cnastorage/kubernetes-anywhere
 ```
 
-### Enter the `kubernetes-anywhere` deployment environment:
+#### Run docker container using `cnastorage/kubernetes-anywhere:latest` image to launch deployment environment:
 
 ```shell
-make docker-dev # Building docker image for first time can take few minutes.
+docker run -it --rm --env="PS1=[container]:\w> " --net=host cnastorage/kubernetes-anywhere:latest /bin/bash
 ```
 
-### Start the deployment wizard:
+#### Start the deployment wizard:
 
```

Comment:
Is this part missing something?
It says "Start the deployment wizard" with 4th level heading but then followed by "Sample config" with 3rd level heading?

Edit: OK I understand now. So the "Sample config" is the content of "Start the deployment wizard"? But the headings are not consistent which can result in some confusing.